content/source/docs/enterprise/api/configuration-versions.html.md

```diff
@@ -8,6 +8,102 @@ sidebar_current: "docs-enterprise2-api-configuration-versions"
 
 -> **Note**: These API endpoints are in beta and are subject to change.
 
+## List Configuration Versions
```

Comment:
Please update this to match the [template](https://github.com/hashicorp/terraform-website/blob/master/content/source/docs/enterprise/api/_template.md).

Reply:
+1!